config/routes.rb

```diff
@@ -53,6 +53,7 @@
   get '/422', to: 'errors#422'
   get '/500', to: 'errors#500'
   get '/auth_error', to: 'errors#auth_error'
+  get "/timeout", to: "errors#timeout"
```

Comment:
This needs to have the name of the controller action in the `to` option, and then lets match the auth_error for the `get`

```suggestion
  get "/timeout_error", to: "errors#timeout_error"
```

The reason the test didn't fail is because by default, even if there is no method in the controller, Rails renders the HTML template with the action name, which is `timeout.html.erb`. The html.erb should also be renamed to `timeout_error.html.erb` to match the controller (and the auth error).

Reply:
Tricky, I would prefer just a failure than a default loading that is non-obvious. Thanks for pointing it out, resolved.